From c744d31e122c84b1471496d0d56d637330eff25d Mon Sep 17 00:00:00 2001
From: neomilium <neomilium@gmail.com>
Date: Sat, 13 Mar 2010 00:27:08 +0100
Subject: [PATCH 3/8] Fix URL construction:
  - if a _space_ is user in the search string (i.e. "grand journal"), requests may failed.

---
 canal.py |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/canal.py b/canal.py
index 4918ce3..a2dffb2 100755
--- a/canal.py
+++ b/canal.py
@@ -77,7 +77,8 @@ def print_err(msg):
 
 def buildURLdico(currentTvShow, nb_video, quality):
     #Get Dom file
-    url= url_canal + currentTvShow
+    url= url_canal + currentTvShow.replace(' ','%20')
+    print_dbg("XML URL: " + url)
     try:
         dom = minidom.parse(urlopen(url))
     except IOError:
-- 
1.6.3.3

